package/gluon-respondd/files/etc/hotplug.d/iface/10-gluon-respondd

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+. /usr/share/libubox/jshn.sh
+. /lib/functions/service.sh
+
+DEVLIST=/var/run/gluon-respondd.devs
+
+ifname_to_dev () {
+	json_load "$(ubus call network.interface.$1 status)"
+	json_get_var dev device
+
+	echo "$dev"
+}
+
+case "$ACTION" in
+	ifdown)
+		sed "/ $INTERFACE$/d" $DEVLIST > $DEVLIST.new
```

Comment:
The specification of sed doesn't guarantee atomicity for `sed -i`. I think most (all?) current implementions use a temporary file for it, but making this explicit is cleaner.